internal/logging/context.go

```diff
@@ -13,7 +13,11 @@ import (
 // already have the root SDK logger and root provider logger setup from
 // terraform-plugin-go tf5server RPC handlers.
 func InitContext(ctx context.Context) context.Context {
-	ctx = tfsdklog.NewSubsystem(ctx, SubsystemHelperSchema, tfsdklog.WithLevelFromEnv(EnvTfLogSdkHelperSchema))
+	ctx = tfsdklog.NewSubsystem(ctx, SubsystemHelperSchema, tfsdklog.Options{
+		// All calls are through the HelperSchema* helper functions
+		tfsdklog.WithAdditionalLocationOffset(1),
+		tfsdklog.WithLevelFromEnv(EnvTfLogSdkHelperSchema),
+	}...)
```

Comment:
Given that the last parameter of `NewSubsystem` is variadic, would this work too?

```suggestion
	ctx = tfsdklog.NewSubsystem(ctx, SubsystemHelperSchema,
		// All calls are through the HelperSchema* helper functions
		tfsdklog.WithAdditionalLocationOffset(1),
		tfsdklog.WithLevelFromEnv(EnvTfLogSdkHelperSchema),
	})
```